notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>delete the GCS bucket?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
The GCS bucket stores experiments output which can be taken away by users.